bootstrap/terraform/main.tf

```
@@ -118,26 +85,45 @@ module "eks_blueprints" {
 #---------------------------------------------------------------
 
 module "eks_blueprints_kubernetes_addons" {
-  source = "github.com/aws-ia/terraform-aws-eks-blueprints//modules/kubernetes-addons?ref=v4.18.1"
+  source = "github.com/aws-ia/terraform-aws-eks-blueprints//modules/kubernetes-addons?ref=v4.22.0"
 
   eks_cluster_id = module.eks_blueprints.eks_cluster_id
 
   # Deploy Crossplane
+  # Default helm chart and providers values set at https://github.com/aws-ia/terraform-aws-eks-blueprints/blob/main/modules/kubernetes-addons/crossplane/locals.tf
   enable_crossplane = true
 
-  crossplane_helm_config = local.crossplane_helm_config
+  #---------------------------------------------------------
+  # Crossplane community AWS Provider deployment
+  #---------------------------------------------------------
+  crossplane_aws_provider = {
+    enable = true
```

I think we need to set the provide config to `aws-provider-config`, if not the name will be `default`
```suggestion
    enable                = true
    provider_config = "aws-provider-config"
```